libraries/botbuilder-runtime-integration-restify/src/index.ts

```diff
@@ -30,32 +37,38 @@ const defaultOptions: Options = {
 export async function start(
     applicationRoot: string,
     settingsDirectory: string,
-    options = defaultOptions
+    options = {} as Options
```

Comment:
We should type this as `Partial<Options>` to make sure we do proper defaulting of `undefined` option values.

Reply:
Sounds good, I didn’t like the look of this code, but that should clean it up nicely. Tak!